flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BigQueryDefaultWriter.java

```diff
@@ -81,6 +105,18 @@ public void write(IN element, Context context) {
         }
     }
 
+    /** Overwriting flush() method for updating Flink Metrics in at-least-once Approach. */
+    @Override
+    public void flush(boolean endOfInput) {
+        super.flush(endOfInput);
```

Comment:
`super` is needed here for sure since you invoking the parent method after over-riding it

Reply:
ACK.